webapp-django/crashstats/crashstats/static/crashstats/js/socorro/timeutils.js

```diff
@@ -0,0 +1,24 @@
+/* global moment */
+
+$(function() {
+    function updateTimes() {
+        $('time.ago').each(function() {
+            var self = $(this);
+            var date = self.attr('datetime') || self.data('date');
+            self
+                .attr('title', self.text())
+                .text(moment(date).fromNow());
+        });
+
+        $('time.in').each(function() {
+            var self = $(this);
+            var date = self.attr('datetime') || self.data('date');
+            self
+                .attr('title', self.text())
+                .text(moment(date).fromNow(true));
+        });
+    }
+
+    updateTimes();
+    setInterval(updateTimes, 60 * 1000);
+});
```

Comment:
It looks like a new one because I moved it to a different location. Let's merge these PRs in order and I'll take care of rebasing them as needed.